GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -136,17 +126,81 @@ public void runImport(File file, String serverBase, Map<String, String> criteria
         Map<Long, Long> columnIndexToQuestionId = processHeader(sheet);
```

Comment:
I took a look into the `processHeader()` method and noticed that you add 1 to the column index when you create this map but I couldn't figure out why.